docker/Dockerfile-slim

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7.3-slim
+LABEL owner="MajorcaDevs"
+COPY ./mdbackup/ /mdbackup 
+COPY ./MANIFEST.in /mdbackup
+COPY ./README.md /mdbackup
+COPY ./setup.py /mdbackup
+WORKDIR mdbackup
+
+RUN mkdir /backups && apt update && apt upgrade -y && \
```

Comment:
Same as above, but only applicable to upgrade. Update is needed to be able to install packages.